pkg/api/v1/types.go

```diff
@@ -462,6 +462,10 @@ type PersistentVolumeSpec struct {
 	// More info: http://kubernetes.io/docs/user-guide/persistent-volumes#recycling-policy
 	// +optional
 	PersistentVolumeReclaimPolicy PersistentVolumeReclaimPolicy `json:"persistentVolumeReclaimPolicy,omitempty" protobuf:"bytes,5,opt,name=persistentVolumeReclaimPolicy,casttype=PersistentVolumeReclaimPolicy"`
+	// Name of StorageClass where this persistent volume belongs. Empty value
+	// means that this volume does not belong to any StorageClass.
+	// +optional
+	ClassName string `json:"className,omitempty"`
```

Comment:
Should be storageClassName per conventions (assuming the resource kind is StorageClass):

https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#naming-conventions

cc @thockin